crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -63,6 +63,27 @@ const CLUSTER_COUNT_MASK: u32 = (1 << CLUSTER_COUNT_SIZE) - 1;
 // The z-slicing method mentioned in the aortiz article is originally from Tiago Sousa's Siggraph 2016 talk about Doom 2016:
 // http://advances.realtimerendering.com/s2016/Siggraph2016_idTech6.pdf
 
+#[derive(Resource)]
+pub struct GlobalClusterSettings {
```

Comment:
I've been really in favor of these kinds of resources for storing available features in general, I think we should have more.